assets/locales/service.cy.toml

```diff
@@ -15,95 +15,98 @@
 
 [FindFactsAndFiguresEnglandWales]
 description = "Find facts and figures heading"
-one = "Find facts and figures about areas in England or Wales"
+one = "Facts and figures for areas in England and Wales"
 
 [GetDataAboutPeopleAndHouseholds]
 description = "Area profiles start page intro"
 one = "Get data about people and households in <a href=\"/areas/E92000001\" data-test=\"a-country-1\">England</a> or <a href=\"/areas/W92000004\" data-test=\"a-country-2\">Wales</a>. You can also view facts and figures for a town, local authority or ward in each country."
 
-[DataIncludes]
-description = "Data includes"
-one = "Data includes"
-
-[HowManyPeopleLiveThere]
-description = "how many people live there"
-one = "how many people live there"
-
-[HowCrowdedItIs]
-description = "how crowded it is"
-one = "how crowded it is"
-
-[PeoplesAverageAge]
-description = "people's average age"
-one = "people's average age"
-
-[PeopleThinkTheirGoodHealth]
-description = "how many people think their general health is good"
-one = "how many people think their general health is good"
-
-[HouseholdsEnglishNotMainLanguage]
-description = "how many households where English is not the main language"
-one = "how many households where English is not the main language"
-
-[HouseholdsWelshMainLanguage]
-description = "how many households where Welsh is the main language"
-one = "how many households where Welsh is the main language (Wales only)"
-
-[HouseholdsWithMortgage]
-description = "how many households are owned with a mortgage, loan or shared ownership"
-one = "how many households are owned with a mortgage, loan or shared ownership"
-
 [OtherCountries]
 description = "Other countries"
 one = "Other countries"
 
-[FindFactsAndFiguresScotland]
-description = "Find facts and figures about areas in Scotland on Scotland’s Census website"
-one = "Find facts and figures about <a href=\"https://www.scotlandscensus.gov.uk/search-the-census#/explore\">areas in Scotland on Scotland’s Census website</a>."
+[OtherCountriesContent]
+description = "Scotland and Northern Ireland have their own agencies who produce official statistics. View facts and figures for areas in:"
```

Comment:
Are we not looking to add welsh content?

Reply:
I think that's coming, i was told to not use google translate myself lol